sample/build.gradle

```diff
@@ -54,6 +54,9 @@ victor {
 
     // Do not generate these densities for SVG assets
     excludeDensities = [ 'ldpi', 'xxxhdpi' ]
+
+    // Do not generate android vector drawables (generate PNGs for the specified densities)
+    generateVectorDrawables = false
```

Comment:
I'd remove this for now, so that CI passes.

Due to the annoying nature of Gradle plugins, I generally update the sample after the next version goes out.